docs/source/methods/dsrg.rst

```diff
@@ -489,6 +489,27 @@ The threshold of considering the BCH expansion converged based on the recursive
 * Type: double
 * Default: 1.0e-12
 
+**DSRG_ADAPTIVE_RSC**
+
+Whether to use an adaptive threshold for the recursive single commutator approximation.
+
+* Type: boolean
+* Default: false
```

Comment:
Does the adaptive rsc change the LDSRG(2) results? If not, and it speed up things, I prefer to make it `true` by default.

Reply:
I think so too. I'll run all of the ldsrg2 tests and see if any of them fail.